.pre-commit-config.yaml

```diff
@@ -61,6 +62,20 @@ repos:
       language: python
       name: Check no tests are ignored
       pass_filenames: false
+    - id: conda-env-sort
+      additional_dependencies: [pyyaml]
+      entry: python scripts/sort_conda_envs.py
+      files: ^conda-envs/
+      language: python
+      name: Sort dependencies in conda envs
+      types: [yaml]
+    - id: pip-to-conda
+      additional_dependencies: [pyyaml]
+      entry: python scripts/generate_pip_deps_from_conda.py
+      files: ^(conda-envs/environment-dev-py38\.yml|requirements-dev\.txt)$
```

Comment:
why not do this for 36 and 37 files too?

Reply:
I was originally thinking that there could be a single conda file which generates the requirements-dev file, but there's no reason why they shouldn't all generate it

Sure, done 👍